src/site/content/en/spaces/space-x1-test/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Title Goes Here Title Goes Here Title Goes Here
+description: We’re learning all about Vertex AI this week as Carter Morgan and Jay Jenkins Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+tags: twitter-space
+hosts:
+  - name: Name 1 here
+    twitter: devnook
+    image: https://pbs.twimg.com/profile_images/519869363344515072/s8vGg8fE_400x400.jpeg
+    host: true
+  - name: Name 2 here
+    twitter: devnook
+    image: https://pbs.twimg.com/profile_images/519869363344515072/s8vGg8fE_400x400.jpeg
+    host: false
+twitter_link: http://twitter.com
+event_date: 2023-02-21
+audio: https://traffic.libsyn.com/secure/thecsspodcast/TCP_CSS_Podcast__Episode_003_v2.0_FINAL.mp3?dest-id=1891556
```

Comment:
Similar to my other comment about hotlinking to Twitter profile images, what's the strategy for storing and linking to Twitter Spaces recordings from a URL we control? How will we guide Hosts to obtain the correct URL when they do this for themselves?

Reply:
We will have url we control for these audio files (stored on drive), this one is just for testing purposes.